freeipa.spec.in

```diff
@@ -255,6 +253,7 @@ Requires: systemd-python
 Requires: %{etc_systemd_dir}
 Requires: gzip
 Requires: oddjob
+Requires: gssproxy >= 0.5.1
```

Comment:
This will change to something like 0.5.2 in any case, do you want the comment in the spec file or in the commit message ?